spec/requests/user_status_controller_spec.rb

```diff
@@ -75,6 +75,19 @@
         expect(response.status).to eq(400)
       end
 
+      it "validates emoji" do
+        put "/user-status.json", params: { description: "girl_with_pink_hair" }
```

Comment:
```suggestion
        put "/user-status.json", params: { description: "invalid_emoji_name" }
```

That's minor change but is makes the code even more explicit, and in case of failure of the test, the error would be even more explicit.